.chloggen/es_preserve_host_name.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: elasticsearchexporter
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Add support for preserving resource attributes when mapping to ECS
```

Comment:
nit: This changelog message is documenting logic and would not make much sense to the end users. I think we should instead add a message specific to the behavior change of `host.name`.